actions/publish_release_images/action.yml

```diff
@@ -20,34 +20,55 @@ inputs:
     description: Path to Dockerbuild context
     type: string
     default: ""
+  platforms:
+    description: Docker platforms to build
+    type: string
+    default: "linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x"
   container_image_name:
     description: Name of the container image
     type: string
-    default: ""
+    default: ${{ github.event.repository.name }}
+  github_token:
+    type: string
+    description: Github Token
 runs:
   using: composite
   steps:
-    - if: inputs.dockerfile_path != ''
-      run: echo "export DOCKERFILE_PATH=${{ inputs.dockerfile_path }}" >> /tmp/tmp-profile
-      shell: bash
-    - if: inputs.container_image_name != ''
-      run: echo "export DOCKER_IMAGE_NAME=${{ inputs.container_image_name }}" >> /tmp/tmp-profile
-      shell: bash
-    - if: inputs.dockerbuild_context != ''
-      run: echo "export DOCKERBUILD_CONTEXT=${{ inputs.dockerbuild_context }}" >> /tmp/tmp-profile
-      shell: bash
-    - run: |
-        current_tag=${GITHUB_REF#refs/*/}
-        touch /tmp/tmp-profile
-        . /tmp/tmp-profile
-        make docker DOCKER_IMAGE_TAG="$current_tag" DOCKER_REPO=${{ inputs.registry }}/${{ inputs.organization }}
-        docker images
-        echo ${{ inputs.password }} | docker login -u ${{ inputs.login }} --password-stdin ${{ inputs.registry }}
-        make docker-publish DOCKER_IMAGE_TAG="$current_tag" DOCKER_REPO=${{ inputs.registry }}/${{ inputs.organization }}
-        make docker-manifest DOCKER_IMAGE_TAG="$current_tag" DOCKER_REPO=${{ inputs.registry }}/${{ inputs.organization }}
-        if [[ "$current_tag" =~ ^v[0-9]+(\.[0-9]+){2}$ ]]; then
-          make docker-tag-latest DOCKER_IMAGE_TAG="$current_tag" DOCKER_REPO=${{ inputs.registry }}/${{ inputs.organization }}
-          make docker-publish DOCKER_IMAGE_TAG="latest" DOCKER_REPO=${{ inputs.registry }}/${{ inputs.organization }}
-          make docker-manifest DOCKER_IMAGE_TAG="latest" DOCKER_REPO=${{ inputs.registry }}/${{ inputs.organization }}
+    - name: Determine latest
+      id: update_latest
+      run: |
+        if [ ${{ startsWith(github.ref, 'refs/tags/v3.') }} ]
```

Comment:
These CI actions are used all over the Prometheus ecosystem. We have v0, v1, etc releases stil.

Maybe we should parameterize this, with something like: `refs/tags/${{ inputs.latest_tag_prefix }}`, 

Then set the default to just `v`, and then when we update Prometheus, we can specify `v3`.

Also, how does this avoid prerelease?